.github/issue_template.md

```diff
@@ -1,5 +1,5 @@
 ## Backgrounds
-TODO: Explain the background of the issue.
+TODO: Explain the background of the issue!
```

Comment:
No, we do not need ! for this. back to `.` please.

Reply:
oh sorry!